### PR TITLE
Bump up golang version to v1.13.4

### PR DESCRIPTION
**Pull request by wy65701436**
Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,6 +128,7 @@ Harbor backend is written in [Go](http://golang.org/). If you don't have a Harbo
 |   1.7    |    1.9.2      |
 |   1.8    |    1.11.2     |
 |   1.9    |    1.12.12    |
+|   1.10   |    1.13.4     |
```
Comment (reasonerjt) [inline CONTRIBUTING.md]: I think we are not doing this for 1.10

**Comment by reasonerjt** [comment]
Did we rebuild the dependent binaries?

File: CONTRIBUTING.md (hunk 1 above)
Comment (wy65701436) [reply CONTRIBUTING.md]: fixed.

**Comment by wy65701436** [comment]
yes, all of the binaries have been upgrade to v1.13.4

**Pull request merged**